gate-web/src/main/groovy/com/netflix/spinnaker/gate/config/CloudEventHandlerConfiguration.java

```diff
@@ -27,11 +28,17 @@ public class CloudEventHandlerConfiguration implements WebMvcConfigurer {
 
   @Override
   public void extendMessageConverters(List<HttpMessageConverter<?>> converters) {
+    converters.add(0, byteArrayHttpMessageConverter());
     converters.add(cloudEventHttpMessageConverter());
   }
 
   @Bean
   public CloudEventHttpMessageConverter cloudEventHttpMessageConverter() {
     return new CloudEventHttpMessageConverter();
   }
+
+  @Bean
+  public ByteArrayHttpMessageConverter byteArrayHttpMessageConverter() {
```

Comment:
I tried removing `@Bean` here and making this method private and the test still passes.

Reply:
the bean is not what is fixing the issues with Swagger. Adding the ByteArrayHttpMessageConverter as the first converter fix it. if you remove the line that add it as the first element you will see the test failing.